woocommerce/woocommerce
# Fix E2E variable product test

### All Submissions:


### Changes proposed in this Pull Request:

This PR supersedes #27543 . Due to WP changes in metabox markup and CSS, Puppeteer in headless mode on MacOS is unable to target selectors in the variation metaboxes. That portion of the test script does work in non-headless mode and in Travis. As an interim solution to allow re-enabling of this test, this PR skips setting variation attributes.

### How to test the changes in this Pull Request:

1. Run E2E tests locally
2. Verify the `can create product with variations` test passes

### Other information:




### Changelog entry

N/A
